src/frontend/src/icp/types/icrc-custom-token.ts

```diff
@@ -13,3 +15,11 @@ export type IcrcCustomTokenWithoutId = UserTokenState & IcTokenWithoutIdExtended
 export type IcrcCustomToken = TokenToggleable<IcToken> & IcrcCustomTokenExtra;
 
 export type OptionIcrcCustomToken = Option<IcrcCustomToken>;
+
+export function isIcrcCustomTokenDisabled(token: IcrcCustomToken): boolean {
```

Comment:
same for this function, that possibly is a mix between zod and an util (plus tests)